app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,9 @@
+module SignUp
+  class CompletionsController < ApplicationController
+    def show; end
```

Comment:
Could we add an analytics call here to track how many users see this page, and pass in the `session[:sp][:loa3]` and `@sp_name` attributes?